.travis.yml

```diff
@@ -17,10 +18,14 @@ matrix:
 install:
   - ./.travis/install.sh
 
-script: tox -e $TOX_ENV
+script:
+  - if [[ -n "${TOX_ENV}" ]]; then tox -e $TOX_ENV; fi
+  - if [[ "$PUSH_DOCS" == "true" ]]; then ./.travis/build_docs.sh; fi
+
+secure: "CvFj8Df5OiDRrW7EsTGhkltdmNlYerx9hH/tSKxiNFVDBUUFaTN7rUr7kWcOKchzerGwk7zjZ4SRXyoSCs+Srht6GZxWHkNROwKpp5Xvf5clbLXbp7GO1X/L5rLgrXpGwtkhgNuHx0X2IUCDHUQAUSumPgZcNFu3emgVxEqabN0="
```

Comment:
Oh yes, should have made a note about this -- this is the content of "travis encrypt 'GIT_NAME="HawkOwl (Automatic)" GIT_EMAIL=hawkowl@atleastfornow.net GH_TOKEN=<my gh token>'". I'll comment that so that recreating it is easier. It populates the env with these variables.